### PR TITLE
[SP-2493] [SP-2454] Options for translate from app

### PR DESCRIPTION
**Pull request by insxne**

### CODE REVIEW

File: app/helpers/translinator/textinator_helper.rb [hunk 1]
```diff
@@ -28,7 +29,7 @@ def default(options)
     end
 
     def handle_error(key, options, exception)
-      Rails.logger.error "Textinator error on fetching key: #{key} with options:#{options.inspect} and params {params.inspect}"\
+      Rails.logger.error "Textinator error on fetching key: #{key} with options:#{options.inspect} and params #{params.inspect}"\
```
Comment (feliguez) [inline app/helpers/translinator/textinator_helper.rb]: Missing Space after options.

File: lib/translinator/textinator.rb [hunk 2]
```diff
@@ -11,11 +22,13 @@ def texts(params)
         end
         JSON.parse(response).with_indifferent_access
       rescue StandardError => exception
-        Rails.logger.error "Textinator error on fetching all texts: with params:#{params.inspect}:"\
+        Rails.logger.error "Textinator error on fetching all texts:\n"\
+        + " with params: #{params.inspect}\n"\
         + " #{exception.message}\n"\
         + exception.backtrace.join("\n")
```
Comment (feliguez) [inline lib/translinator/textinator.rb:28]: Unneeded double quotes for `exception.backtrace.join("\n")`

Comment (insxne) [inline lib/translinator/textinator.rb:28]: Es necesario, si no, no hace salto de linea!

File: app/helpers/translinator/textinator_helper.rb (hunk 1 above)
Comment (insxne) [inline app/helpers/translinator/textinator_helper.rb]: Ja!

File: app/controllers/translinator/textinator_callback.rb [hunk 3]
```diff
@@ -1,11 +1,10 @@
 module Translinator
   module TextinatorCallback
-    def translate!(search = {})
-      @tl = Textinator.texts({
-        application: Rails.application.class.parent_name.downcase,
-        countryCode: defined?(I18n) ? I18n.locale : :cl,
-        url: request.original_fullpath
-      }.merge(search)) if Translinator::Config.enabled?
+    def translate!(options = {})
+      return true if !Translinator::Config.enabled?
```
Comment (feliguez) [inline app/controllers/translinator/textinator_callback.rb]: Prefer `unless` with negative booleans.

File: app/helpers/translinator/textinator_helper.rb [hunk 4]
```diff
@@ -1,12 +1,13 @@
 module Translinator
   module TextinatorHelper
     def tl(key, options = {})
+      options.merge(@tl_app_vars.call)
```
Comment (feliguez) [inline app/helpers/translinator/textinator_helper.rb]: Add `!`operator if you want to modify object `options`

Comment (feliguez) [inline app/helpers/translinator/textinator_helper.rb]: Ja!

**Comment by insxne** [comment]
Issue Jira: SP-2454

**Pull request merged**